### PR TITLE
Js edits

### PR DESCRIPTION
**Pull request by sargonas**
- updated install instructions
- merged in Gene's seeder rewrite from the wiki

### CODE REVIEW

File: docs/getting-started/installation.md [hunk 1]
```diff
@@ -8,13 +8,17 @@ import Tabs from '@theme/Tabs';
...
 
-However, if you intend to run a Chia full node on a server and connect to it programmatically using the [RPC interface](/rpc), the best method would be to install and run Chia on the command line.
+- If you intend to run a dedicated Chia full node on a server and connect to it programmatically using the [RPC interface](/rpc), the best method would be to install and run Chia via the command line on a proper server environment.
 
-Finally, if you plan on making contributions to the source code or doing [Chialisp](https://chialisp.com) development, we recommend installing Chia from source. This gives the highest level of flexibility.
+- If you inded to do [Chialisp](https://chialisp.com) development or build projects that leverage Chia, you have the options of either using an installer (the recomended pattern), or installing from source.
```
Comment (danieljperry) [inline docs/getting-started/installation.md:17]: indend -> intend

Comment (danieljperry) [inline docs/getting-started/installation.md:17]: recomended  -> recommended

Comment (sargonas) [reply docs/getting-started/installation.md:17]: and to think I dumped this into a separate tool to spell check TWICE...

thanks!

File: docs/getting-started/installation.md [hunk 2]
```diff
@@ -376,9 +385,11 @@ The config file under the `config` subdirectory. Its name is `config.yaml`, and
 
 It is possible to configure the `CHIA_ROOT` environment variable to another location. A common use for this would be to switch it to `~/.chia/testnet` to have a separate config for the testnet.
 
-## CLI Setup
+- - - - -
 
-Using the CLI gives greater and more precise control over the various Chia services such as the full node. For a more details on the commands, read the [CLI Reference](/cli).
+## CLI
+
+Using the CLI gives greater and more precise control over the various Chia services such as the full node. As of 1.8.2, when installing from an installer or package CLI commands will be automatically added to your path for Windows and Linux. For a more details on the commands, read the [CLI Reference](/cli).
```
Comment (danieljperry) [inline docs/getting-started/installation.md:392]: For a more details -> For more details

File: docs/getting-started/installation.md (hunk 1 above)
Comment (sargonas) [reply docs/getting-started/installation.md:17]: corrected in updated PR

Comment (sargonas) [reply docs/getting-started/installation.md:17]: corrected in updated PR

File: docs/getting-started/installation.md (hunk 2 above)
Comment (sargonas) [reply docs/getting-started/installation.md:392]: corrected in updated PR

**Review by hoffmang9 — DISMISSED** [review]
lgtm

File: docs/guides/seeder-user-guide.md [hunk 3]
```diff
@@ -23,93 +21,133 @@ dig dns-introducer.chia.net
...
+
+## Being included in config.yaml
+
+There are a couple of criteria we all look for when adding a seeder to the initial config file.
+
+You must have an ICANN registered domain name. Your seeder host must support IPv6 and IPv4.
+
+We ask that you commit to a monthly uptime of 99.9% which is available enough to be reliable but also leaves flexibility to not need to run a cluster and gives you time for a reboot once in a while to keep up with things like security patches. We will be monitoring all of the seeders listed in the most recent version of `initial-config.yaml`. We highly recommend that you enable monitoring of your seeder as well. We are heavy users of [Uptime Robot](https://uptimerobot.com/) but anything similar will do.
+
+Being a known community member - the better you're known (pseudonymously is fine) - the more likely you are to be added. You should be on [Keybase](https://keybase.io/team/chia_network.public) and we will need to know your Keybase handle as you will be added to a seeder operator's closed channel.
```
Comment (danieljperry) [inline docs/guides/seeder-user-guide.md:149]: Let's not add a new keybase reference for poor Roy to remove later :)

Comment (hoffmang9) [reply docs/guides/seeder-user-guide.md:149]: Having Keybase here is intentional. We don't want to do security conversations on Discord

Comment (sargonas) [reply docs/guides/seeder-user-guide.md]: I've updated the KB instructions in that case

**Review by hoffmang9 — APPROVED** [review]
looks even better

**Pull request merged**